content/concept-checks.md

```diff
@@ -76,7 +76,10 @@ All Checks start the same way, so **Steps 1 and 2** are the same.
 
 ## Availability of Checks
 
-Checks require `rippled` v0.90.0 or later and have been enabled on the production XRP Ledger since 2018-03-15.
+Checks require `rippled` v0.90.0 or later. The Checks amendment was enabled for voting on the production XRP Ledger on 2018-03-22 in this [EnableAmendment pseudo-transaction](https://ripple.com/build/transactions/#enableamendment): [`EC0D90F114D4F3C599FEABCAF47DCF0CAC29A272D1EF6C9BEF7F420819540F04`](https://xrpcharts.ripple.com/#/transactions/EC0D90F114D4F3C599FEABCAF47DCF0CAC29A272D1EF6C9BEF7F420819540F04).
```

Comment:
Let's not say "enabled for voting", because you don't need a transaction for a transaction to become open for voting. What we really mean is it gained support of a majority of trusted validators. If you look at the Flags of the raw transaction, you'll see the value 65536, also known as [`tfGotMajority`](https://ripple.com/build/transactions/#enableamendment-flags). In other words, the comment just below this is slightly misleading because this _is_ the GotMajority transaction.